client/app/lib/activity/sidebar/sidebarmachinesharepopup.coffee

```diff
@@ -238,5 +238,5 @@ module.exports = class SidebarMachineSharePopup extends KDModalView
 
   TITLES =
     'shared machine' : 'wants to share their VM with you. <span class="footnote">* Rejecting this will destroy any existing collaboration sessions with this machine.</span>'
-    'collaboration'  : 'wants to collaborate with you on their VM.'
+    'collaboration'  : 'wants to collaborate on with you on their VM.'
```

Comment:
that's correct copy, the one you need to change is not this one. Apply this `wants to collaborate with you on their VM.` to this https://github.com/koding/koding/issues/9980